packages/tupaia-web/src/features/Map/MapOverlaysLayer/MapOverlaysLayer.tsx

```diff
@@ -3,18 +3,45 @@
  *  Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
  */
 
-import React from 'react';
+import React, { useEffect } from 'react';
+import { useParams } from 'react-router';
+import { useMap } from 'react-leaflet';
 import { LegendProps } from '@tupaia/ui-map-components';
 import { useMapOverlayMapData } from '../utils';
 import { PolygonLayer } from './PolygonLayer';
 import { MarkerLayer } from './MarkerLayer';
+import { useEntity } from '../../../api/queries';
+
+// When entity is selected, zoom to entity
+const useZoomToEntity = () => {
+  const { projectCode, entityCode } = useParams();
+  const { data: entity } = useEntity(projectCode, entityCode);
+  const map = useMap();
+
+  useEffect(() => {
```

Comment:
Just put a comment here that we are replacing the map-ui-components leaflet map positioning